apps/cowswap-frontend/src/modules/ordersTable/pure/OrdersTableContainer/OrderRow/index.tsx

```diff
@@ -203,43 +203,83 @@ export function OrderRow({
     return orders.every((order) => order.status === OrderStatus.CANCELLED)
   }
 
-  const renderFillsAt = () => (
-    <>
-      {getIsFinalizedOrder(order) ? (
-        order.executionData.partiallyFilled || order.status === OrderStatus.FULFILLED ? (
+  const renderFillsAt = () => {
```

Comment:
It must be a separate component

Reply:
The whole component needs to be decomposed into small pure components.
Please, try to keep component files up to ~200 lines of code.